docs/blog/20230401-Daeraxa-AprUpdate.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Community Update
+author: Daeraxa
+date: 2023-04-01
+category:
+  - news
+  - log
+tag:
+  - update
+---
+
+Another dose of our regular monthly community update!
+
+<!-- more -->
+
+# Welcome to the April Community Update (don't worry, no fooling going on here)!
+
+Hi everyone and welcome to the April instalment of our monthly update that keeps keeps you informed about what is going on in the background of Pulsar, work in progress, community contributions and more!
```

Comment:
```suggestion
Hi everyone and welcome to the April installment of our monthly update that keeps keeps you informed about what is going on in the background of Pulsar, work in progress, community contributions and more!
```

Reply:
This one's a UK/US difference, so maybe we flip a coin? ;-)

Reply:
Yeah but I guess its meant to be from "the team" rather than me specifically so it reverts to en-US rules, didn't even know that was a difference...